app/controllers/library_entries_controller.rb

```diff
@@ -13,6 +13,28 @@ def authorize_operation(operation)
     end
   end
 
+  def issues
+    authenticate_user!
+    missing_library_data = {
+      rating_ids: [],
+      reaction_ids: [],
+      attributes_ids: []
+    }
+
+    library_entries = LibraryEntry.includes(
+      :media_reaction,
+      media: :media_attribute_votes
+    ).where(user: user)
+
+    library_entries.each do |entry|
+      missing_library_data[:rating_ids] << entry.id if entry.rating.nil? && entry.status.completed?
+      missing_library_data[:reaction_ids] << entry.id if entry.media_reaction.nil?
+      votes = entry.media.media_attribute_votes.select { |mv| mv.vote == :unvoted }
+      missing_library_data[:attributes_ids] << entry.id unless votes.empty?
+    end
```

Comment:
This is gonna be really really slow (30s+ slow for many libraries), you probably need to do this via joins instead